script/refunder.js

```diff
@@ -43,7 +59,27 @@ async function main() {
   // Waiting for the transaction to be mined
   const new_receipt = await new_tx.wait();
   // The transaction is now on chain!
-  console.log(`Mined transaction (see in its corresponding block explorer): ${new_receipt.transactionHash}`);
+  console.log(
+    `Mined transaction (see in its corresponding block explorer): ${new_receipt.transactionHash}`
+  );
+}
+
+async function getEthFlowOrderBytes(tx, ethflow_address) {
+  if (tx.to.toLowerCase() === ethflow_address.toLowerCase()) {
+    return tx.data.substring(10);
```

Comment:
Ideally we'd have 1 way of doing things to make the code simpler. Whatever new method we end up with should work for the simple case too.